L1Trigger/TrackFindingTracklet/src/Tracklet.cc

```diff
@@ -644,149 +632,168 @@ void Tracklet::setFitPars(double rinvfit,
   fpgatrack_ = std::make_unique<Track>(makeTrack(l1stubs));
 }
 
-std::string Tracklet::trackfitstr() {
-  string stubid0 = "111111111";
-  string stubid1 = "111111111";
-  string stubid2 = "111111111";
-  string stubid3 = "111111111";
+const std::string Tracklet::layerstubstr(const unsigned layer) const {
+  assert(layer <= 5);
```

Comment:
super minor but can the 5 be a constant? (like N_DISK if that's what the 5 means)

Reply:
correction: can be "layer < N_LAYER" ? where N_LAYER==6 (didn't read properly first...)

Reply:
Done.